html/Callbacks/OAuth/NoAuth/Logout.html/Default

```diff
@@ -0,0 +1,10 @@
+<%ARGS>
+$URL => undef
+</%ARGS>
+<%INIT>
+# There are two Default callbacks. We want the one with $URL.
```

Comment:
Since RT 4.2, the callback that could update $URL  is changed to "ModifyLoginRedirect", not "Default" any more.

Reply:
See above re 4.4.0.